python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -308,6 +311,15 @@ def last_completion_time(self) -> Optional[float]:
         """Optional[float]: Timestamp representing the last time this sensor completed an evaluation."""
         return self._last_completion_time
 
+    @public
+    @property
+    def last_start_time(self) -> Optional[float]:
+        """Optional[float]: Timestamp representing the last time this sensor was started. Can be
```

Comment:
ah particularly when paired with last_completion_name the symmetry would make me think both times referred to a tick